pages/docs/tutorials/getting-started/slackbot-websocket-tutorial.md

```diff
@@ -0,0 +1,340 @@
+---
+title: Create an AsyncAPI Document for a SlackBot with WebSockets
+description: In this tutorial, you'll learn how to create an AsyncAPI document designed for a Slack application that operates in Socket Mode using the WebSockets protocol.
+weight: 80
+---
+
+## Introduction
+In this tutorial, you will learn how to generate an AsyncAPI document designed for a Slack application that operates in Socket Mode. The aim is to help you grasp a real-world application of AsyncAPI with the WebSocket protocol.
+
+Consider a scenario where you are in charge of maintaining a highly active Slack workspace. You want an easy way to keep track of the popular messages across all the channels but doing this manually would be a difficult task. To simplify this process, you’re going to build a Slackbot called `Heart-Counter-Slack-Bot` that actively monitors reactions added to a message and determine its popularity by counting the reactions of the “heart” emoji.
+
+Here’s a visual representation of how `Heart-Counter-Slack-Bot` should work: 
+
+```mermaid
+sequenceDiagram
+participant Heart-Counter-Slack-Bot
+participant Slack Server (Socket Mode)
+participant User
+
+
+Heart-Counter-Slack-Bot ->>Slack Server (Socket Mode): Connect
+Note right of Heart-Counter-Slack-Bot: Establish WebSocket connection
+
+
+Slack Server (Socket Mode)->> Heart-Counter-Slack-Bot : Handshake
+Note left of Slack Server (Socket Mode): Establish Socket Mode connection
+
+Slack Server (Socket Mode)->> Heart-Counter-Slack-Bot : Sends "hello" event
+Note left of Slack Server (Socket Mode): Confirms successful connection
+
+
+
+
+
+User-->>Slack Server (Socket Mode): Adds emoji to a message.
+
+
+Slack Server (Socket Mode)->>Heart-Counter-Slack-Bot: Sending "reaction_added" payload
+
+
+Note left of Heart-Counter-Slack-Bot: Event received
```

Comment:
your app basically interacts with two servers, one slack http server that is used to receive the `ticket` and the `app_id`, and uses these two to establish a connection to the websocket server. please reflect this in your diagram.